promtimer/promtimer.py

```diff
@@ -217,6 +250,8 @@ def main():
     parser.add_argument('--dont-open-browser', dest='dont_open_browser', default=False,
                         action='store_true',
                         help='don\'t open browser tab automatically on start')
+    parser.add_argument("--stats-archive-path", dest='stats_archive_path',
```

Comment:
might be nice to have "backup" somewhere in the --arg itself to distinguish it (but also might be a bit long winded, whatever you prefer 👍 )

Reply:
Agreed